app/models/manageiq/providers/redhat/infra_manager/inventory/strategies/v4.rb

```diff
@@ -60,6 +61,11 @@ def collect_clusters
       connection.system_service.clusters_service.list
     end
 
+    def collect_cluster(uuid)
+      cluster = connection.system_service.clusters_service.cluster_service(uuid).get
+      [cluster]
```

Comment:
@borod108 OK, will follow this approach. We need to work on consistent coding style. It will be the first occurrence.

@masayag Clusters do not change that often. There are many calls which ignore 404 and we may want to consider it to fix for all. I would like to provide separate PR to handle it.